Source/DataSources/CzmlDataSource.js

```diff
@@ -1015,6 +1015,7 @@ define([
         processPacketData(Cartesian3, billboard, 'alignedAxis', billboardData.alignedAxis, interval, sourceUri, entityCollection);
         processPacketData(Boolean, billboard, 'show', billboardData.show, interval, sourceUri, entityCollection);
         processPacketData(VerticalOrigin, billboard, 'verticalOrigin', billboardData.verticalOrigin, interval, sourceUri, entityCollection);
+        processPacketData(Boolean, billboard, 'sizeInMeters', billboardData.sizeInMeters, interval, sourceUri, entityCollection);
```

Comment:
That requires a pull request into `czml-writer` to update to the schema (the guid is auto-generated).  There's actually quite a few things that are missing from the schema/guide (whole objects even).  I wouldn't worry about this until we are ready to invest the time in cleaning up `czml-writer`